ui-library-quickstart-teams-interop-meeting-chat/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "ui-library-quickstart-teams-interop-meeting-chat",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@azure/communication-react": "1.8.0-beta.1",
+    "@fluentui/react": "8.101.0",
+    "@testing-library/react": "^11.2.7",
+    "@testing-library/user-event": "^12.8.3",
+    "@types/jest": "^26.0.24",
```

Comment:
I think 
`"@testing-library/react": "^11.2.7",
    "@testing-library/user-event": "^12.8.3",
    "@types/jest": "^26.0.24",`
    
    can be removed

Reply:
right thank you!